scripts/noaa/gpcc_spi/download.py

```diff
@@ -0,0 +1,79 @@
+# Copyright 2022 Google LLC
```

Comment:
We should have a wrapper shell script or at least README instructions that ties everything together.